scm/etc/scripts/forcing_file_common.py

```diff
@@ -1,5 +1,7 @@
 #!/usr/bin/env python
 
+from __future__ import print_function
```

Comment:
this is of course only required for Python2, but it won't hurt with Python3.

Reply:
Yep, I thought that we were trying to make any python scripts compatible with 2 AND 3.